### ISSUE TITLE
Incorrect topicset/topicref processing in FO template

### ISSUE THREAD

**Issue by mironovalexey** [dita-ot/dita-ot#2648]
First level topicrefs within topiset container are excluded from FO/PDF processing.

As far as I understand [this selector](https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.pdf2/xsl/fo/root-processing.xsl#L225) should be more accurate. Something like this
`<xsl:for-each
select="opentopic:map/*[contains(@class, ' map/topicref ') and not(contains(@class, ' mapgroup-d/topicset '))] |
opentopic:map/*[contains(@class, ' mapgroup-d/topicset ')]/*[contains(@class, ' map/topicref ')]">`

Otherwise 
`<map>
<topicset id="xxx">
  <topicref href="..."/>
</topiset>
</map>` 
will lead to incorrect FO tree and broken PDF.

**Comment by robander** [comment]
This really isn't related to the `<topicset>` element, and the fix shouldn't make use of the `topicset/@class` token. That element is really just ... a `topicref` that has a required ID, with no other special processing (it just has the rather subjective extra meaning of "a set of topics"). I can get the same failure with a single root `<topicref>` element that does not have `@href`; I can get fix the failure with the root `<topicset>` by adding `@href` to it.

So, I think all it takes to break the PDF is:
```
<map>
  <topicref>
    <topicref href="goodtopic.dita"/>
  </topicref>
</map>
```

Attaching a version of that test case here with one root `<topicref>` (no `@href`) and 2 simple sub-topics.
[2648.zip](https://github.com/dita-ot/dita-ot/files/988007/2648.zip)

**Comment by robander** [comment]
Related (and same root cause): when I have a map like this, I don't get the build failure, but only the first topic ends up in the PDF -- the second topic is in the TOC / Bookmarks but nowhere else:
```
<map xml:lang="en-us"><title>Grouped group group</title>
  <topicref href="2648a.dita"/>
  <topicref>
    <topicref href="2648b.dita"/>
  </topicref>
</map>
```

**Comment by jelovirt** [comment]
Fixed by #2683
